IDGPredict/FacetPredict.h

```diff
@@ -200,7 +206,7 @@ class FacetPredict {
     double uvwr2 = uvw[0] * uvw[0] + uvw[1] * uvw[1] + uvw[2] * uvw[2];
     if (uvw[2] > _maxW && uvwr2 <= _maxBaseline * _maxBaseline) {
       Flush();
-      _maxW *= 1.5;
+      _maxW = std::max(uvw[2], _maxW * 1.5);
```

Comment:
Same question here. Why did this change?

Reply:
The idea is that we are increasing _maxW if it is smaller than uvw[2]. When multiplying _maxW by 1.5 it can still be smaller than uvw[2]. If that's the case, we are increasing _maxW to uvw[2] now.